### DISCUSSION TITLE
Question about structure

### DISCUSSION THREAD

**Discussion by alexcoca** [pyscaffold/pyscaffoldext-dsproject#22] [Q&A]
Hi,

I was wondering why are you putting the project under `src/pyscaffoldext/dsproject/` as opposed to `src/dsproject`?

Many thanks,

**Comment by FlorianWilhelm** [comment]
Hi @alexcoca, I converted your issue to a question as this seems more suitable for me, I hope you are fine with this. Regarding your question. It's all about avoiding name clashes by using a namespace package `pyscaffoldext`. The idea is that all extensions to PyScaffold live in the namespace `pyscaffoldext` to avoid any conflicts with existing Python packages, e.g. `django`. Does this make sense?
